### ISSUE TITLE
Make Documents in FAP Documents Tab keyboard Accessible

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1658]
You currently cannot select documents in the Documents tab of the FAP page with just a keyboard.
